ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/SubmissionValidator.scala

```diff
@@ -231,5 +231,9 @@ object SubmissionValidator {
     Envelope.enclose(value).toByteArray
 
   private[validator] def bytesToStateValue(value: RawBytes): DamlStateValue =
-    Envelope.openStateValue(value).right.get
+    Envelope
+      .openStateValue(value)
+      .fold(message => {
```

Comment:
Can you remove the curly braces to simplify, please?

Reply:
Done.